packages/admin/cms-admin/package.json

```diff
@@ -46,6 +49,7 @@
         "final-form-arrays": "^3.0.2",
         "final-form-set-field-touched": "^1.0.0",
         "graphql-tag": "^2.0.0",
+        "immutable": "^4.3.0",
```

Comment:
Why did you decide to add the Draft.js conversion parts to `@comet/cms-admin`, and not `@comet/admin`? Also, they shouldn't be in src/common, but rather in src/rte or src/blocks/richText, if they are rich text block related only.

Reply:
had no reason, have moved it now